docs/compass/02-01-components.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Components
+type: Architecture
+---
+
+![Components](./assets/components.svg)
```

Comment:
something is wrong in that diagram, Kyma wraps compass, mind blow. In architecture, maybe we should first focus of ways how compass component could be deployed, with two modes, single cluster, and master and slaves clusters.

Reply:
Changed the diagrams so that Compass is placed inside the `Cluster`, not Kyma.